### ISSUE TITLE
Problem upgrading DAQ after git pull

### ISSUE THREAD

**Issue by pisuke** [faucetsdn/daq#17]
I've don a git pull to upgrade DAQ to the latest code on the master branch.
Then issued the following commands

```
sudo cmd/clean
sudo bin/clean_dev
sudo bin/setup_base
sudo bin/setup_dev
cmd/run -s 
```

So far, so good.
Then, running

```
sudo cmd/build 
```

the build script execution stops with this output:

Loading build configuraiton from local/system.conf
Build docker/Dockerfile.aardvark into daq/aardvark, log to build/docker_build.aardvark...
E: Unable to locate package tcpdump
E: Unable to locate package strace
Retry 2 of cmd apt-get failed with exit code 100
E: Unable to locate package net-tools
E: Unable to locate package iproute2
E: Unable to locate package iputils-ping
E: Unable to locate package tcpdump
E: Unable to locate package strace
Retry 3 of cmd apt-get failed with exit code 100
The command '/bin/sh -c $AG update && $AG install net-tools bash iproute2 iputils-ping tcpdump strace' returned a non-zero code: 100
Build failed, see build/docker_build.aardvark for complete log.

Here is the output of the build/docker_build.aardvark log file

Sending build context to Docker daemon  105.4MB
Step 1/5 : FROM ubuntu:xenial
 ---> 52b10959e8aa
Step 2/5 : WORKDIR /root
 ---> Running in bf8c666f4121
Removing intermediate container bf8c666f4121
 ---> 5de4177e6b7c
Step 3/5 : COPY bin/retry_cmd bin/
 ---> 8fa513a51e35
Step 4/5 : ENV AG="bin/retry_cmd apt-get -qqy --no-install-recommends -o=Dpkg::Use-Pty=0"
 ---> Running in 8d8618616145
Removing intermediate container 8d8618616145
 ---> ef1ddd9e0a78
Step 5/5 : RUN $AG update && $AG install net-tools bash iproute2 iputils-ping tcpdump strace
 ---> Running in 8fb940bc53e6
W: Failed to fetch http://archive.ubuntu.com/ubuntu/dists/xenial/InRelease  Temporary failure resolving 'archive.ubuntu.com'
W: Failed to fetch http://archive.ubuntu.com/ubuntu/dists/xenial-updates/InRelease  Temporary failure resolving 'archive.ubuntu.com'
W: Failed to fetch http://archive.ubuntu.com/ubuntu/dists/xenial-backports/InRelease  Temporary failure resolving 'archive.ubuntu.com'
W: Failed to fetch http://security.ubuntu.com/ubuntu/dists/xenial-security/InRelease  Temporary failure resolving 'security.ubuntu.com'
W: Some index files failed to download. They have been ignored, or old ones used instead.
E: Unable to locate package net-tools
E: Unable to locate package iproute2
E: Unable to locate package iputils-ping
E: Unable to locate package tcpdump
E: Unable to locate package strace
Retry 1 of cmd apt-get failed with exit code 100
E: Unable to locate package net-tools
E: Unable to locate package iproute2
E: Unable to locate package iputils-ping
E: Unable to locate package tcpdump
E: Unable to locate package strace
Retry 2 of cmd apt-get failed with exit code 100
E: Unable to locate package net-tools
E: Unable to locate package iproute2
E: Unable to locate package iputils-ping
E: Unable to locate package tcpdump
E: Unable to locate package strace
Retry 3 of cmd apt-get failed with exit code 100
The command '/bin/sh -c $AG update && $AG install net-tools bash iproute2 iputils-ping tcpdump strace' returned a non-zero code: 100

Hope this can be useful to get it fixed.
Thanks,
Francesco

**Comment by grafnu** [comment]
I just tried it and it worked for me (and on the CI build), so it's either
transitory (try it again!), or there could be some bad state in your build
container (not sure how or what), or something to do with your local
system.  It seems to be a very common problem
<https://www.google.com/search?q=docker+temporary+failure+resolving+%27archive.ubuntu.com%27&oq=docker+tempo&aqs=chrome.0.0j69i57j0l4.3470j0j1&sourceid=chrome&ie=UTF-8>,
but no know "easy" solution... just a bunch of different things that worked
for different people.  So, might try one of those.  Or, maybe stop/start
docker. Or reboot. (Yah, we've regressed to that level of solution...)
